Tracy calling InitThread() trigger error

Importing ztracy and calling InitThread causes following error:

```txt
$ zig build -Dztracy-enable=true run-game-of-life
.\deps\zig-gamedev\libs\ztracy\src\ztracy.zig:321:10: error: container '.ztracy.cimport:242:15' has no member called '___tracy_init_thread'
        c.___tracy_init_thread();
         ^
````

--- a/libs/ztracy/src/ztracy.zig
+++ b/libs/ztracy/src/ztracy.zig
@@ -41,7 +41,6 @@ const tracy_stub = struct {
         }
     };
 
-    pub inline fn InitThread() void {}
     pub inline fn SetThreadName(name: [*:0]const u8) void {
         _ = name;
     }
@@ -317,9 +316,6 @@ const tracy_full = struct {
         }
     }
 
-    pub inline fn InitThread() void {
-        c.___tracy_init_thread();
-    }
     pub inline fn SetThreadName(name: [*:0]const u8) void {
         c.___tracy_set_thread_name(name);
     }

